nan_callbacks.h

```diff
@@ -101,7 +101,7 @@ static const int kIndexPropertyQueryIndex =      5;
 static const int kIndexPropertyFieldCount =      6;
 
 template<typename T, typename P>
-P *GetWrapper(T needle) {
+static inline P *GetWrapper(T needle) {
```

Comment:
Just inline apparently led to SIGSEGV. The problem is likely that clang includes static locals many times. The haystack exists to enforce singleton behavior for callbacks, which is necessary since the wrappers are never freed. The wrappers, in turn, are necessary, because SetInternalFieldPointer only accepts data pointers, not function pointers.